crates/client/src/client.rs

```diff
@@ -968,6 +971,24 @@ impl FuelClient {
 
         Ok(proof)
     }
+
+    pub async fn relayed_transaction_status(
+        &self,
+        id: &Bytes32,
```

Comment:
Should we also use RelayedTransactionId here?

Reply:
The type we are using for `transaction_status` is `TxId` which is owned by `fuel-vm`, but is just an alias of `Bytes32`, so I just left this... Maybe we should consider changing both? I'm not too opinionated.

I think the best maybe is to define a type in `/client` that is the alias of `Bytes32`. I like the idea of having the client own its types, but I'm not too worried either way on this one.

Reply:
Yeah alias is probably the only safe way at this point to avoid breaking changes on apis that are already present. For the new API being added here we could use the RelayedTransactionId though since it wouldn't break anything.